lib/experimental/Widgets/Content/TasksList/index.tsx

```diff
@@ -1,4 +1,5 @@
-import { Circle, InProgressTask } from "@/icons"
+import { Badge } from "@/components/Information/Badge"
+import { DottedCircle, InProgressTask } from "@/icons"
 import { useCallback } from "react"
 
 type TaskStatus = "in-progress" | "due" | "no-due"
```

Comment:
What means due, no-due?

I understand "in-progress" but the other two are not clear, maybe we can rename it?